### ISSUE TITLE
Command Line Can't Be Found

### ISSUE THREAD

**Issue by mufid** [mtuchowski/mdspell#1]
Hi, i try to run `mdspell` but seems like the CLI is not working

```
vmuser@vm1:~/repo $ gem install mdspell
Successfully installed mdspell-0.1.2
Parsing documentation for mdspell-0.1.2
Done installing documentation for mdspell after 0 seconds
1 gem installed
vmuser@vm1:~/repo $ mdspell
mdspell: command not found

```

**Comment by mtuchowski** [comment]
Turns out I forgot to add proper gemspec command. Thanks. Fixed in `0.1.3` - already on RubyGems.

**Comment by mufid** [comment]
Thanks! But then i get different error

```
vmuser@vm1:~/repo/hackathon$ mdspell hackathon-sebelumnya.md
/home/vmuser/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- mixlib/cli (LoadError)
        from /home/vmuser/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/gems/mdspell-0.1.3/lib/mdspell/cli.rb:1:in `<top (required)>'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/gems/mdspell-0.1.3/lib/mdspell.rb:2:in `require_relative'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/gems/mdspell-0.1.3/lib/mdspell.rb:2:in `<top (required)>'
        from /home/vmuser/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from /home/vmuser/.rvm/rubies/ruby-2.2.1/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/gems/mdspell-0.1.3/bin/mdspell:7:in `rescue in <top (required)>'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/gems/mdspell-0.1.3/bin/mdspell:2:in `<top (required)>'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/bin/mdspell:23:in `load'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/bin/mdspell:23:in `<main>'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
        from /home/vmuser/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
v
```

**Comment by mtuchowski** [comment]
Fixed in `0.1.5` - update from RubyGems.
